service-base/src/main/java/org/eclipse/hono/service/management/tenant/AbstractTenantManagementHttpEndpoint.java

```diff
@@ -310,11 +310,6 @@ protected final boolean isValidRequestPayload(final JsonObject payload) {
      *       or the JsonArray contains elements that are not of type {@link JsonObject}.
      */
     protected final void addNotPresentFieldsWithDefaultValuesForTenant(final JsonObject checkedPayload) {
-        if (!checkedPayload.containsKey(RegistryManagementConstants.FIELD_ENABLED)) {
-            logger.trace("adding 'enabled' key to payload");
-            checkedPayload.put(RegistryManagementConstants.FIELD_ENABLED, Boolean.TRUE);
-        }
-
```

Comment:
is there a particular reason why you want to remove this?

Reply:
This step is already done in `updatePayloadWithTenantId` in the same class.

Reply:
Adding the enabled field is already done a few line before, when calling `Util.getRequestPayload(JsonObject)` which already add the field.